src/test/java/com/sampleweb/journey/SifatTests.java

```diff
@@ -0,0 +1,69 @@
+package com.sampleweb.journey;
+
+import com.sampleweb.BaseTest;
+import com.sampleweb.framework.PageNavigator;
+import com.sampleweb.pages.CarAndMotorbikePage;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+// A wild lenny appears ( ͡° ͜ʖ ͡°)
+
+public class SifatTests extends BaseTest {
+
+    private PageNavigator navigator = new PageNavigator();
+    private CarAndMotorbikePage carAndMotorbikePage;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        carAndMotorbikePage = navigator.navigateToPage(driver, CarAndMotorbikePage.PATH, CarAndMotorbikePage.class);
+        carAndMotorbikePage.selectOptionFromDropdown();
+
+    }
+
+    @Test (groups = "regression")
+    public void carNavigationPageTest() {
+        assertThat(carAndMotorbikePage.isLoaded(), is(true));
+    }
+
+    @Test (groups = "regression")
+    public void carUrlNavigationPageTest() {
+        assertThat(carAndMotorbikePage.isLoaded(), is(true));
+    }
+
+   /* @Test (groups = "regression")
```

Comment:
Could delete if you have no need for it.

Reply:
Done